oarepo_oidc_einfra/resources.py

```diff
@@ -95,14 +97,23 @@ def accept_invitation(self) -> Response:
 
         request_id = decrypt(request.view_args["request_id"])
```

Comment:
It would be safer to generate a random one-time login token for this purpose & associate it with a certain request record, rather than directly use a more persistent `request_id`, that might get leaked over e.g. OS/requests APIs